### DISCUSSION TITLE
suggestion

### DISCUSSION THREAD

**Discussion by iliaoke** [community/community#74619] [Mobile]
希望可以添加检查fork项目的更新，一键拉取最新的源代码，这个功能好像只有在网页版上有

**Comment by eliperkins** [comment]
Hey @liaoke01 , thanks for this feedback. Forgive me for any mistakes, as I'm using a translation service to help you out. It sounds like you want a button to update a forked repo from its upstream repo.

This isn't something that's currently on the Mobile team's radar, so I've created a tracking issue on our internal repo for this feature request. I'll let you know when this has been prioritized.

For my understanding, can I ask a couple of questions to help clarify?

- What workflow do you have that would benefit from updating a forked repo right from GitHub Mobile? That is, how are you using your fork today where it needs to be up-to-date?
- How would you expect to find out that your forked repo is out-of-date from the upstream repo? By navigating to it? A notification? An email?

Thanks for helping us improve GitHub Mobile!

**Reply by iliaoke** [reply]
Thank you for your reply. To be precise, I want GitHub Mobile to be a complete replacement for GitHub Web on mobile and in any of its features. For now, I feel like he's more of an aid than a replacement. For example, it doesn't have the ability to delete repositories. As for one-click check for updates, and the advantage of forking upstream is that for some personal projects, you don't need to modify anything, you just need to keep the upstream source code consistent, such as when deploying a web page with vercel, so that I don't have to go to the web version to operate, which can improve my efficiency, after all, you know that the experience of github web version on mobile is not good
